### PR TITLE
Create page for linking to terraform provider documentation

### PR DESCRIPTION
**Pull request by rileybrook**
Create page for linking to terraform provider documentation

### CODE REVIEW

File: docs/terraform-provider-documentation [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Terraform provider documentation
+
+Save time on managing Airbyte Cloud and collaborate on Airbyte configuration changes with your teammates. Airbyte's Terraform provider is based on our new [Airbyte API}(! The Terraform provider is available for Airbyte Cloud customers today (& OSS in Q3). Check out our guide for getting started with our Terraform provider(Save time on managing Airbyte Cloud and collaborate on Airbyte configuration changes with your teammates. Airbyte's Terraform provider is based on our new [Airbyte API](https://reference.airbyte.com/reference/start)! The Terraform provider is available for Airbyte Cloud customers today (& OSS in Q3). Check out our guide for [getting started with Airbyte's Terraform provider](https://reference.airbyte.com/reference/using-the-terraform-provider).
```
Comment (terencecho) [inline docs/terraform-provider-documentation:3]: looks like this part is unintentional `[Airbyte API}(!` and there same doc repeats itself

**Review by terencecho — CHANGES_REQUESTED** [review]

File: docs/terraform-provider-documentation (hunk 1 above)
Comment (rileybrook) [reply docs/terraform-provider-documentation:3]: Whoops! Thanks

**Review by terencecho — APPROVED** [review]

File: docs/terraform-provider-documentation [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Terraform provider documentation
```
Comment (marcosmarxm) [inline docs/terraform-provider-documentation:1]: If you want to display the page in docs.airbyte.io you must add the documentation in sidebar.js inside docusarus page.

Comment (marcosmarxm) [reply docs/terraform-provider-documentation:1]: I think the documentation can be placed inside manage-airbyte folder

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Review by terencecho — APPROVED** [review]

**Review by terencecho — CHANGES_REQUESTED** [review]
sorry, previous approval on accident, build still not working

**Review by marcosmarxm — APPROVED** [review]

**Review by terencecho — APPROVED** [review]

**Pull request merged**